sources/packages/web/nginx.conf

```diff
@@ -0,0 +1,30 @@
+user  nginx;
+worker_processes  1;
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+events {
+  worker_connections  1024;
+}
+http {
+  include       /etc/nginx/mime.types;
+  default_type  application/octet-stream;
+  log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log  /var/log/nginx/access.log  main;
+  sendfile        on;
+  keepalive_timeout  65;
+  server {
+    listen       8080;
```

Comment:
when nginx listens on 8080, it serves on port 80 and 443 (http and https). Should we consider changing the openshift route?

![image](https://github.com/bcgov/SIMS/assets/54600590/e8c5dc84-0a63-457f-ade2-f3bb9be143dd)

Reply:
I belive that we may need to update the Docker file to have something like `EXPOSE ${PORT}:8080`.

Reply:
@dheepak-aot I'm going to change to use the env var: `"$PORT"`

Reply:
EXPOSE ${PORT} is missing in dockerfile. is that expected?

Reply:
Also are we not changing the port in make file. ?
![image](https://github.com/bcgov/SIMS/assets/54600590/0be6e99f-a97e-45dd-af48-f923302475ed)

Reply:
Changing the port is not needed as it'll use the env variable `PORT`. I'm trying to change as minimum as possible. The `EXPOSE` is not needed either.

Reply:
![image](https://github.com/bcgov/SIMS/assets/78114138/682d0283-23d2-412c-b84d-ae5dbb5c7f9d)

Reply:
@dheepak-aot I had to set `ENV PORT 3030` in Dockerfile and did some changes in Dockerfile to replace the variable ${PORT} in default.conf because the ubi8 image works different from the image used in Dockerfile.dev.